linear_operator/test/linear_operator_test_case.py

```diff
@@ -1247,3 +1247,12 @@ def test_svd(self):
             for arg, arg_copy in zip(linear_op.representation(), linear_op_copy.representation()):
                 if arg_copy.requires_grad and arg_copy.is_leaf and arg_copy.grad is not None:
                     self.assertAllClose(arg.grad, arg_copy.grad, **self.tolerances["svd"])
+
+    def test_to_double(self):
+        # test if the linear_op is still functional after convert to double
+        linear_op = self.create_linear_op()
+        try:
+            linear_op = linear_op.to(torch.float64)
+            linear_op.numpy()
```

Comment:
Should we also test that the dtype was properly updated? 

Also, we probably want to overwrite this for the interpolated operator to verify that the index tensors were not moved to floating point type

Reply:
Make sense, I will include test to check if dtype is updatedm, and check if index tensors are still integers for the interpolated operator case.